README.md

```diff
@@ -0,0 +1,147 @@
+This is logically necessary for the protocol to identify which party broadcasted the commitment transaction and potentially punish them if the other party provides provides proof it has been revoked (i.e. knows the revocation key).
+
+Wouldn't it be nice if we could identify the broadcasting party without assigning them different transactions?
+Riard first considered this problem in [8] and proposed that you could identify the party using _witness asymmetry_ i.e. both parties broadcast the same transaction but have different witnesses. 
+Unfortunately, the solution they propose is rather convoluted.
+
+More recently in [1], Aumayr et al. introduced a much more elegant witness asymmetric solution using adaptor signatures.
+Instead of being assigned different transactions, the parties are assigned different adaptor signatures as witnesses for the _same_ transaction.
+The adaptor signatures force the party broadcasting a commitment transaction to reveal a "publishing secret" to the other party.
+The honest party can then use this publishing secret along with knowledge of the usual revocation secret to punish the malicious party for broadcasting an old commitment transaction.
+
+In the paper this idea is combined with "punish-then-split" mechanism similar to the original eltoo[8] proposal and therefore it unfortunately inherits the issue with staggering time-locks.
+BOLT 3 [4] notes this problem as design motivation:
+
+> The reason for the separate transaction stage for HTLC outputs is so that HTLCs can timeout or be fulfilled even though they are within the to_self_delay delay. Otherwise, the required minimum timeout on HTLCs is lengthened by this delay, causing longer timeouts for HTLCs traversing the network.
+
+More in depth commentary by Towns can be found in [2] and they propose a solution to the eltoo proposal in [3].
+
+In short, the problem is that it creates a risk for the party that needs to fulfill an HTLC with the secret in time.
+The only known way of accounting for this risk is to increase the difference between the time-locks on each hop.
+There could be situations where this trade off makes sense but it seems like it undesirable for a general purpose payment channel network.
+
+# Proposal
+
+I propose a channel protocol which steals the main idea from Aumayr et al. while retaining the time-lock precedence of BOLT 3 transactions.
+That is, absolute time-locks are settled before relative time-locks to avoid having to account for relative time-locks when calculating absolute time-locks.
+
+The guiding principal is to keep commitment transactions symmetric but to use the asymmetric knowledge of the parties to simulate the way lightning works right now.
+The main benefits of this seem to be:
+
+- It is more elegant as there are half the number of possible transactions. I expect this will follow through to reduced implementation complexity and maybe make it easier to explain as well.
```

Comment:
Yes it should simplify onchain transaction parser and thus increase their reliablity.